.ci/install_kubernetes.sh

```diff
@@ -36,6 +36,11 @@ if [ "$ID" == "fedora" ]; then
 	bash -f "${cidir}/configure_crio_for_kata.sh"
 fi
 
+# Get fedora version
+if [ "$ID" == "fedora" ]; then
+	fedora_version=$(grep VERSION_ID /etc/os-release | cut -d '=' -f2)
```

Comment:
I think you can do this in the `elif` that handles centos and fedora

`elif [ "$ID" == "centos" ] || [ "$ID" == "fedora" ]; then`

Reply:
@devimc changes applied